_release/bat/base_bat/base_bat_test.go

```diff
@@ -311,6 +327,180 @@ func TestDeleteAllInstances(t *testing.T) {
 	}
 }
 
+//Launch an instance created from an image with a customized cloud init
+//
+//Download cirros image, add it to the image service using a custome cloudinit file
+//create a workload, launch an instance of this workload and connect via SSH
+//using the custom user and password
+//
+//instance should accept connection with specified user and password and
+//reply with the user name
+
```

Comment:
I'm not sure the test-cases tool will pick up the comments if there is a newline separating the comments from the test function.  You can check with

cd _release/bat/base_bat
test-cases -format tap

And check you see the description of your test case in the tap output.